### DISCUSSION TITLE
How can i adjust markdown interdocument links when i use markdowninput?

### DISCUSSION THREAD

**Discussion by SpotBR** [Witiko/markdown#379] [Q&A]
I have a documentation in markdown, with a tree folder structure. So now we can convert it to any front end we want.

We would like to use your package, to convert multiple files and folders to a pdf. 

So we want to use markdownInput for this. We expect that the package can generate the index of markdowninput

But, there is inter document links on our environment, and we need to convert it to internal links. 

There is an better way to do our try?

R we smart?

@Witiko

**Comment by SpotBR** [comment]
can anyone help me?

@Witiko

**Reply by Witiko** [reply]
I am sorry, but it's not clear to me what you are asking. Can you provide some code examples to illustrate your issue?
